ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/ClassLoadInvoker.java

```diff
@@ -480,9 +483,8 @@ private ClassLoadContext createVariablesExecutionContext(
             Set<QuotedIdentifier> requiredImports = new HashSet<>();
             String variableType = importsManager.extractImportsFromType(typeSymbol, requiredImports);
             this.newImports.put(variableName, requiredImports);
-
-            // Create a global variable
-            GlobalVariable globalVariable = new GlobalVariable(variableType, variableName,
+            String kind = typeSymbol.typeKind().getName();
```

Comment:
Again, directly use the enum itself.

Reply:
Updated in 3fc03c0dee2d7091929456e76b603ab332cb786e